### DISCUSSION TITLE
req.multipart0 not responding/working

### DISCUSSION THREAD

**Discussion by timfemey** [kartikk221/hyper-express#90] [Q&A] [answered]
When using req.multipart() i get no response or data passed into the  callback argument. And i also had an error before with req.multipart() like i said in #86 , after sometime without any code changes it doesnt just respond

**Comment by kartikk221** [comment, marked as answer]
I'm not quite sure why that would be the case. Could you provide a code snippet which can properly recreate the problem?

**Reply by timfemey** [reply]
Thanks for the reply, Its working right now since I upgraded to V6.2.5 and I also set headers property in the `req.multipart` Busboy Config as 
`headers: { "content-type":req.headers["content-type"] }` instead of just `headers: req.headers`.

awesome project by the way,  really appreciate the work.
